### ISSUE TITLE
NVDA announces dialog heading once but speech viewer shows twice when dialog has element with tablist role

### ISSUE THREAD

**Issue by sbanerjee97** [nvaccess/nvda#12077]
If a dialog has an element with tablist role and the dialog is opened and one its tabs is clicked, the speech viewer shows the
dialog header twice.
<b>Note: Occurs in firefox and chrome</b>
[wcag.txt](https://github.com/nvaccess/nvda/files/5987950/wcag.txt)

### Steps to reproduce:
1. Open NVDA and turn on its speech viewer.
2. Open a dialog having an element with tablist role and click on one of its tabs.

### Actual behavior:
You will notice that NVDA announces dialog heading once but speech viewer shows it twice.
![image](https://user-images.githubusercontent.com/68270378/108051078-74040280-7070-11eb-8b8f-44d190a1d28b.png)

### Expected behavior:
NVDA should announce and show dialog heading in speech viewer once.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA 2020.3
#### Windows version:
Windows 10
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by feerrenrut** [comment]
For anyone confused about this behavior, they should note that text sent to speechviewer is shown "instantaneously", and is not removed, and currently has no ability to show cancelled text. 

Speech on the other hand can be cancelled (by user, or by NVDA) moments after initially started, often too quick to hear anything at all. This can lead to the situation described here.

There certainly improvements we could make to the speech viewer, both in terms of being a more accurate reflection of what is actually spoken, but also capturing other information about the way speech is delivered.

**Comment by Adriani90** [comment]
cc: @hwf1324, @cary-rowen you might be interested in knowing about this one.
